openapi.json

```diff
@@ -2736,6 +2786,387 @@
+        }
+      }
+    },
+    "/users/{id}/attachments": {
+      "get": {
+        "summary": "Lists attachments in user",
+        "description": "Lists attachments belonging to a user by their ID.",
+        "tags": [
+          "Attachments"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "User's ID",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "example": 42
+          },
+          {
+            "name": "unassigned",
+            "in": "query",
+            "description": "If set, returns unassigned attachments, that are available for assigning to a transaction.",
+            "required": false,
+            "schema": {
+              "type": "integer"
+            },
+            "example": 1
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/Attachment"
+                  }
+                }
+              }
+            }
+          },
+          "403": {
+            "$ref": "#/components/responses/403"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        }
+      },
+      "post": {
+        "summary": "Create attachment in user",
+        "description": "Creates an attachment belonging to the user by their ID.",
+        "tags": [
+          "Attachments"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "User's ID",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "example": 42
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "title": {
+                    "type": "string",
+                    "description": "The title of the attachment. If the title is blank or not provided, the title will derived from the file name.",
+                    "example": "Invoice for taxi"
+                  },
+                  "file_name": {
+                    "type": "string",
+                    "description": "The file name of the attachment",
+                    "example": "taxi.png"
+                  },
+                  "file_data": {
+                    "type": "string",
+                    "format": "base64",
+                    "description": "the base64-encoded contents of the source file. The supported file types are png, jpg, pdf, xls, xlsx, doc, docx.",
+                    "example": "Base64.strict_encode64(where-my-file-is-kept)"
```

Comment:
This example isn't valid Ruby nor is it pseudo-code really. I think we should just remove the example in this case.